### PR TITLE
Service page tabs become active correctly

### PR DESCRIPTION
**Pull request by kennyt**
# Before
px

# After
px

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @Poltergeist

File: src/js/pages/ServicesPage.js [hunk 1]
```diff
@@ -42,14 +42,23 @@ var ServicesPage = React.createClass({
...
   updateCurrentTab: function () {
     let routes = this.context.router.getCurrentRoutes();
     let currentTab = routes[routes.length - 1].name;
     // `services-page` tab also contains routes for 'services-details'
     if (currentTab === 'services-detail') {
       currentTab = 'services-page';
     }
-    if (currentTab != null) {
+
+    if (currentTab == null) {
```
Comment (jfurrow) [inline src/js/pages/ServicesPage.js]: Could we combine this with the `if` statement on line 53? So `if (currentTab === 'services-detail' || currentTab == null) {`...

Comment (kennyt) [inline src/js/pages/ServicesPage.js]: done

**Comment by pierlo-upitup** [comment]
Thanks!

**Comment by pierlo-upitup** [comment]
Would be great to have some integration tests for this, though! Out of scope?

**Comment by kennyt** [comment]
@pierlo-upitup alright i added some integration tests.

px

px

**Pull request merged**